packages/textfield/src/textfield.css

```diff
@@ -56,7 +56,25 @@ textarea {
     overflow: hidden;
 }
 
+:host([grows]) #textfield:after {
+    grid-area: unset;
+    min-block-size: calc(
+        var(
+                --mod-text-area-min-block-size,
+                var(--spectrum-text-area-min-block-size)
+            ) +
+            var(
+                --mod-textfield-focus-indicator-gap,
+                var(--spectrum-textfield-focus-indicator-gap)
+            ) * 2
+    );
+}
+
 :host([grows]) #sizer {
```

Comment:
The padding rules for this need to be updated to match those applied to the `<textarea>` roughly:
```
padding-block-end: var(--mod-textfield-spacing-block-end,var(--spectrum-textfield-spacing-block-end) );
    padding-block-start: var(--mod-textfield-spacing-block-start,var(--spectrum-textfield-spacing-block-start) );
    padding-inline: var(--mod-textfield-spacing-inline,var(--spectrum-textfield-spacing-inline) );
```

Reply:
Looks so much nicer, here!